docs/catalogue/cat_network-data-manager.md

```diff
@@ -43,48 +48,41 @@ section [Upload metadata](cat_network-data-manager.md#upload-metadata) to the Da
 
 <sup>*Figure 1. Tables in a Network’s staging area in the Data Catalogue.*</sup>
 
-#### *Releases* sheet
+#### *TargetDataDictionaries* sheet
 
-Versioning of the network's data model is defined in the *Releases* sheet. Columns with an asterisk (\*) after their
-name are mandatory.
+Versioning of the network's data model is defined in the *TargetDataDictionaries* sheet. Columns with an asterisk (\*) 
+after their name are mandatory.
 
 | *Column name* | *Description* | *Remarks* |
 | --- | --- | --- |
-| resource \* | Name of the network. | e.g LifeCycle, LONGITOOLS or ATHLETE |
-| version \* | Version of the data release. | e.g. 1.1 |
-
-<sup>*Table 1. Description of the columns that can be filled out for Releases. \* = mandatory*</sup>
+| resource \* | Name of the network's model. | e.g LifeCycle_CDM, LongITools_CDM or ATHLETE_CDM |
+| version \* | Version of the data dictionary. | e.g. 1.1 |
 
-![Figure 2. Example of Releases filled out in Excel.](../img/cat_target-release-sheet.png ':size=150')
-
-<sup>*Figure 2. Example of Releases filled out in Excel.*</sup>
+<sup>*Table 1. Description of the columns that can be filled out for TargetDataDictionaries. \* = mandatory*</sup>
 
 #### *TargetTables* sheet
 
```

Comment:
The TargetTables sheet also contains the columns keywords and numberOfRows, not listed in the table below

Reply:
I have deleted these from the sheet.